components/collator/src/options.rs

```diff
@@ -260,6 +260,9 @@ pub enum BackwardSecondLevel {
 
 /// Options settable by the user of the API.
 ///
+/// With the exception of reordering (BCP47 `kr`), options that can by implied by locale are
+/// also settable via the API.
```

Comment:
In the previous sentence "API" refers to the general collator API, but here you mean `CollatorOptions` explicitly, right?
```suggestion
/// also settable via [`CollatorOptions`].
```